### ISSUE TITLE
[Question] [Playwright Test] Run new browser instance for each test with --worker. 

### ISSUE THREAD

**Issue by karanshah-browserstack** [microsoft/playwright#8981]
When running multiple tests with -workers, we can see that the number of browsers spawned is the same as the number of workers specified. Is there any way we can spawn a new browser for each test instead of new pages?

**Comment by dgozman** [comment]
@karanshah-browserstack Tell us more about your usecase please. Why do you want a separate browser? Running multiple browsers is less efficient and usually does not give you any benefits.

**Comment by karanshah-browserstack** [comment]
@dgozman We are trying to run this in remote mode(i.e client running on one machine and the server is running on a different machine) and users can spawn multiple browser connections in parallel. If different tests are using the same browser connection, the users find it unintuitive to locate a specific test within a bunch of tests that have run in the same browser. 

It would ideal if there was an option to specify(an optional parameter) that every playwright test runs in a different browser session.

**Comment by karanshah-browserstack** [comment]
@dgozman Workers are recreated when a test fails to keep pristine environment, instead of doing this just on failures, we need this to happen for each and every spec. Currently we aren't able to do this as workers are reused. Can you please help if this is possible through a workaround or we need a config similar to "reuseWorker" as provided?

**Comment by pavelfeldman** [comment]
#### Why was this issue closed?

We are prioritizing the features based on the upvotes, recency and activity in the issue thread. It looks like this issue only has a handful of upvotes, has not been touched recently and/or we lack sufficient feedback to act on it. We are closing issues like this one to keep our bug database maintainable. Please feel free to open a new issue and link this one to it if you think this is a mistake.
